src/only-export-components.ts

```diff
@@ -79,6 +82,11 @@ export const onlyExportComponents: TSESLint.RuleModule<
       ? new Set(allowExportNames)
       : undefined;
 
+    const createContextMethodsSet = new Set([
+      ...createContextMethods,
+      "createContext",
+    ]);
```

Comment:
Here the number of entries will be too low to have benefits from a Set!
```suggestion
    const createContextMethods = [
      ...createContextMethods,
      "createContext",
    ];
```